### PR TITLE
fix: radio group font size fix

### PR DESCRIPTION
**Pull request by ekateriinal**
Fixed font-size of cvi-radio-group. Issue #62
 @certainlyakey

### CODE REVIEW

**Review by certainlyakey — APPROVED** [review]
Thank you @ekateriinal!

File: libs/ui/src/lib/feedback/feedback/feedback.component.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <cvi-ng-radio-group [appearance]="appearance"
                     [title]="title"
-                    [formControl]="radioButtonsControl">
+                    [formControl]="radioButtonsControl"
+                    style="--radio-group-font-size: 12px">
```
Comment (siimveskilt) [inline libs/ui/src/lib/feedback/feedback/feedback.component.html:4]: styles should not be written in html

**Review by siimveskilt — CHANGES_REQUESTED** [review]

File: libs/ui/src/lib/feedback/feedback/feedback.component.html (hunk 1 above)
Comment (certainlyakey) [reply libs/ui/src/lib/feedback/feedback/feedback.component.html:4]: @siimveskilt I'd say in this case it is OK. We discussed this with @ekateriinal, and turning the font size into a CSS variable that can be set outside of the component seemed like a good option. We already use this kind of external styling API in `cvi-tooltip` or `cvi-textfield`.

Comment (siimveskilt) [reply libs/ui/src/lib/feedback/feedback/feedback.component.html:4]: it is a non-functional requirement that there are no styles used directly inside html files

**Review by siimveskilt — APPROVED** [review]

**Pull request merged**